main.html

```diff
@@ -5,18 +5,104 @@
 		<meta charset="UTF-8">
 		<title>pokedex</title>
 		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<!-- make sure to link to your css file -->
+		<link rel="stylesheet" type="text/css" href="main.css">
 	</head>
 
 	<body>
-		<!-- include a header tag -->
+		<header>
+      <div id=header-navigation>
+        <button>Dropdown
+          </button>
+        </div>
+			<h3 id="header-title">PokeDex</h3>
+      <div id=header-navigation><img src="assets/icons/pokeball_icon.png"></div>
+		</header>
+
 
 		<!-- depending on how you structure your html, you should have 3 section tags -->
 			<!-- 1) search box -->
+		<div id="search-bar">Search Pokemon
+	  	</div>
 			<!-- 2) selected pokemon: picture, property and description of pikachu -->
-			<!-- 3) other pokemon pictures in 2x6 format -->
-
+    <section>
+      <div>
+        <div id="pokemon-pikachu">
+        	<img src="assets/images/pikachu.png"> 
+        </div>
+		<div id="pokemon-description">
+			<div id="pokemon-number">
+				<h2>No. 001 Pikachu</h2>
+				</div>
+			<div id="pokemon-detail">
+				<p>- Electric type</p>
+				<p>- Gentle temperment</p>
+				<p>- Grasslands</p>
+				</div>
+			</div>
+		</div>
+		<div id="pokemon-story">
+			<p> Pikachu can be seen napping in a bright sunlight. It has two red dots on its cheeks. By practicing HTML and CSS, Pikachu can learn confusion. Pikachu likes to steal computer chargers from sleeping college students. </p>
+			</div>
+      </section>
+    <!-- 3) other pokemon pictures in 2x6 format -->
+	  <section>
+		<div id="pokemon-grid">
+			<div id="pokemon-image">
+				<img src="assets/images/whatisthat.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/abra.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/vulpix.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/nidorino.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/clafary.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/charmeleon.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/charmander.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/firething.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/jigglypuff.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/horsea.png">
+				</div>
+			<div id="pokemon-image">
+				<img src="assets/images/pidgey.png">
+				</div>
+			<div id="pokemon-image">
```

Comment:
pokemon-image should be used as a class instead of an id. an id should only be used in a single unique case. because you have multiple pokemon-image cases (ie for each image) a class would be more appropriate.